hbase-rest/pom.xml

```diff
@@ -432,24 +436,13 @@
             <exclusions>
               <exclusion>
                 <groupId>com.sun.jersey</groupId>
-                <artifactId>jersey-core</artifactId>
+                <artifactId>jersey-server</artifactId>
               </exclusion>
             </exclusions>
           </dependency>
         </dependencies>
       </dependencyManagement>
       <dependencies>
-        <dependency>
-          <groupId>org.apache.hadoop</groupId>
-          <artifactId>hadoop-mapreduce-client-core</artifactId>
-          <scope>test</scope>
-          <exclusions>
-            <exclusion>
-             <groupId>com.google.guava</groupId>
-             <artifactId>guava</artifactId>
-            </exclusion>
-          </exclusions>
-        </dependency>
```

Comment:
This change related?

Reply:
Yes, the dependency on jersey is a mess... If we do not add these changes in the pom of hbase-rest, the UTs in hbase-rest will fail with ClassNotFound or AbstractMethod...